pallets/avn/src/tests/tests.rs

```diff
@@ -131,13 +58,13 @@ mod calling_is_primary_validator_for_ethereum {
         let mut ext = ExtBuilder::build_default().with_validators().as_externality();
         ext.execute_with(|| {
             let expected_primary = 1;
-            let eth_index_before: u8 = get_index_based_on_operation_type(&OperationType::Ethereum);
+            let eth_index_before: u8 = AVN::get_primary_collator();
```

Comment:
where is this defined?

Reply:
Its the getter for the primary collator storage index.